localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -636,7 +656,16 @@ void NDTScanMatcher::callbackSensorPoints(
     ndt_pose_with_covariance_pub_->publish(result_pose_with_cov_msg);
   }
 
-  publishTF(ndt_base_frame_, result_pose_stamped_msg);
+  tf2_broadcaster_.sendTransform(
```

Comment:
This is the same process as the `publish_tf` function, so please delete this line.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/1757/commits/b255b43cde6c73522ef6cd4a66bd2af58cdffedb